README.md

```diff
@@ -32,16 +32,20 @@ Google Groups, IRC #activeadmin and Gitter are not actively monitored.
 
 ## Want to contribute?
 
-The [contributing guide][contributing]
-is a good place to start. If you have questions, feel free to ask.
+If you want to contribute through code or documentation, the [contributing
+guide][contributing] is a good place to start. If you have questions, feel free
+to ask.
 
-## Want to support us?
+If you want to contribute economically, you can help funding the project
+through a [Tidelift subscription][tidelift]. By buying a Tidelift subscription
+you make sure your whole dependency stack is properly maintained, while also
+getting a comprenhensive view of outdated dependencies, new releases, security
+alerts, and licensing compatibility issues.
 
-Subscribe to [Tidelift][tidelift] to support Active Admin and get licensing assurances and timely security notifications.
+You can also support us with a weekly tip via [Liberapay].
 
-You can support us with a weekly tip via [Liberapay][liberapay.com].
-
-[![Support via Liberapay][liberapay_button]][liberapay_donate]
+Finally, we have an [opencollective page] where you can become a backer or
```

Comment:
I'm not familiar with this link markdown style syntax. I just want to make sure we show "opencollective" as "Open Collective" which is what they use.

EDIT: I think this would be it?

```suggestion
Finally, we have an [Open Collective][opencollective page] where you can become a backer or
```